packages/components/src/components/button/button.jsx

```diff
@@ -61,14 +62,24 @@ const Button = ({
             {...props}
         >
             {icon && <div className='dc-btn__icon'>{icon}</div>}
-            {text && !(is_loading || is_submit_success) && (
-                <span className={classNames('dc-btn__text', classNameSpan)}>
-                    {text[0].toUpperCase() + text.substr(1)}
-                </span>
-            )}
+            {text &&
+                !(is_loading || is_submit_success) &&
+                ((typeof props.rendertext === 'function' &&
+                    props.rendertext(text[0].toUpperCase() + text.substr(1))) || (
+                    <Text size='xs' weight='bold' align='center' className={classNames('dc-btn__text', classNameSpan)}>
+                        {text[0].toUpperCase() + text.substr(1)}
```

Comment:
Could we use `text-transform: capitalize;` to achieve this? Or could we add a prop to `Text` for this or just pass the style through?

Reply:
It is not capitalize, it is uppercase at the first word and lowercase for the rest.